src/Microsoft.Identity.Web.TokenAcquisition/LoggingEventId.cs

```diff
@@ -29,6 +29,9 @@ internal static class LoggingEventId
         public static readonly EventId CredentialLoadAttemptFailed = new EventId(406, "CredentialLoadAttemptFailed");
         public static readonly EventId UsingSignedAssertionFromCustomProvider = new EventId(407, "UsingSignedAssertionFromCustomProvider");
 
+        // MergedOptions EventIds 408+
+        public static readonly EventId AuthorityConflict = new EventId(408, "AuthorityConflict");
```

Comment:
500+? 400+ is CCA stuff.


```suggestion
        // MergedOptions EventIds 500+
        public static readonly EventId AuthorityConflict = new EventId(500, "AuthorityConflict");
```